packages/ui/tailwind-utils-config/components/tag.ts

```diff
@@ -52,7 +52,8 @@ function createTagVariantStyles(variant: 'outline' | 'secondary'): CSSRuleObject
 
       // ICON STYLES
       '.cn-tag-icon': {
-        color: `var(--cn-set-${theme}-${isOutline ? 'surface-text' : 'soft-text'})`
+        color: `var(--cn-set-${theme}-${isOutline ? 'surface-text' : 'soft-text'})`,
+        opacity: 'var(--cn-opacity-70)'
```

Comment:
@ankormoreankor why we are changing opacity of icons to 0.7 always? 

cc: @cxramina